Fix attempting to install libfmt.a

After the move to cmake for the underlying build system, `make install` can install extraneous artifacts, e.g. see this logged failure from installing `libfmt.a`:

```
-- Install configuration: "Release"
-- Installing: /usr/local/lib/libfmt.a
CMake Error at build/_deps/fmt-build/cmake_install.cmake:46 (file):
  file INSTALL cannot copy file
  "(snip)/yosys-slang/build/third_party/slang/lib/libfmt.a"
  to "/usr/local/lib/libfmt.a": Success.
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -26,6 +26,7 @@ include(FetchContent)
 
 # The log will show "-- Using remote fmt library", but that is misleading,
 # as no network access will be done. (Our first declaration of `fmt` wins.)
+option(FMT_INSTALL OFF)
 FetchContent_Declare(
     fmt
     EXCLUDE_FROM_ALL

